server/go/README.md

```diff
@@ -0,0 +1,30 @@
+# Asherah Server - Go
+
+## Running the server
+This assumes mysql is running on localhost and a preexisting asherah database
```

Comment:
can point here a link to the `Metastore.md` doc file, which can help in setting up the metastore.